metadata/src/main/java/org/apache/kafka/controller/PartitionChangeBuilder.java

```diff
@@ -51,32 +50,46 @@ public static boolean changeRecordIsNoOp(PartitionChangeRecord record) {
         return true;
     }
 
+    /**
+     * Election types.
+     */
+    public enum Election {
+        /**
+         * Perform leader election to keep the partition online or if the preferred replica is in the ISR.
```

Comment:
The second half of this sentence reads a little odd, how about:
```suggestion
         * Perform leader election to keep the partition online. Elect the preferred replica if it is in the ISR.
```